Add video entity.md (#358)
hellohaptik/docs_haptik: docs/bot-builder/basic/entities.md

**Attachment** entity is meant for collecting attachments from the user message.

![Attachment Entity](assets/Video1.png)

There are these three types of attachments allowed -

- Image
- Document (pdf, docx, xls, csv)

> - Video

Reviewer: Can we mention it here that this is only supported for Whatsapp?

Author: I have written that below, we can add it here too

> - Video

Author: ```suggestion
- Video (Only for Whatsapp)
```

![Attachment Entity](assets/Video2.png)

You would need to create a Button HSL for letting the user upload the image and document. 

![Attachment Example on Bot](assets/AttachmentHSL.png)

> In the above example, if the user is not giving a valid PANCARD image then the **entity reprompt message** would be asking the user for a valid image.

Reviewer: That is a confusing statement. We don't validate the images that are uploaded by the user.

If user upload AADHAR card image when asked for PANCARD, we won't be able to figure that out(except for human assistance)

Author: check now

> In the above example, if the user is not giving a valid PANCARD image then the **entity reprompt message** would be asking the user for a valid image.

Author: ```suggestion
In the above example, if the user is not giving an image then the **entity reprompt message** would be asking the user for a valid input.
```

> **Video** attachment type is only for Whatsapp bots as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message -

Reviewer: Can we highlight the Whatsapp piece? Get's lost while reading it.

Where do we mention the PII information details? Where the uploaded video will be deleted from our end in 3 days?
@RandiveSushant @medhaanand

> **Video** attachment type is only for Whatsapp bots as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message -

Author: ```suggestion
**Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message -
```

> "Please wait while we process your video message."

Reviewer: Is this needed to be specified explicitly? Adds confusion saying will this message be returned instead of Bot says?

Author: Yes this needs to be specified here as or else it will increase confusion if not present. People might not know where is this message coming if they have not written it there

Author: changed the content to explain it better

For video entity to be captured on a Whatsapp bot, we need to enable the settings from Business Manager as shown below -

> ![Attachment Entity](assets/Video3.png)

Reviewer: This looks good 👍

**Attachment** entity is meant for collecting attachments from the user message.

![Attachment Entity](assets/Video1.png)

There are these three types of attachments allowed -

- Image
- Document (pdf, docx, xls, csv)
- Video (Only for Whatsapp)

![Attachment Entity](assets/Video2.png)

You would need to create a Button HSL for letting the user upload the image and document. 

![Attachment Example on Bot](assets/AttachmentHSL.png)

> In the above example, if the user is not giving an image then the **entity reprompt message** would be asking the user for a valid input.

Reviewer: `if the user is not giving an image` -> `if the user is not uploading an image file` maybe?

Reviewer: `user for a valid input` maybe rephrase this.

Reviewer: I understand these are not your changes, so you can skip them for now if it's too much to change.

> In the above example, if the user is not giving an image then the **entity reprompt message** would be asking the user for a valid input.

Author: ```suggestion
In the above example, if the user is not uploading an image file then the **entity reprompt message** would ask the user for a valid input.
```

> **Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message -

Author: ```suggestion
**Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message along with your predefined Bot Says messages -
```

> **Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message along with your predefined Bot Says messages -

Reviewer: `we will be able to upload videos there` -> `the user will be able to upload videos there` maybe?

> **Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message along with your predefined Bot Says messages -

Author: ```suggestion
**Video** attachment type is only for **Whatsapp bots** as the user will be able to upload videos only on Whatsapp. Once you successfully upload the video, you will get an automated message along with your predefined Bot Says messages -
```
